meson.build

```
@@ -0,0 +1,111 @@
+project('opensc', 'c',
+    version: '0.23.165'
```

note that for the final version, the version will be best to be moved away to some separate file/script to make sure we do not have to update it on several places for a release. I have only experience with libcacard, where this is done with git-version-gen from autotools, but I believe there will be other/better ways.

https://gitlab.freedesktop.org/spice/libcacard/-/blob/master/meson.build#L2
https://gitlab.freedesktop.org/spice/libcacard/-/blob/master/build-aux/git-version-gen

---

If it's acceptable to update to a minimum required meson >=0.57 then you can do this with:
```suggestion
    version: files('VERSION.txt')
```

and meson will read that file as a version file. For scripts, the libcacard approach seems fine.

From local testing, this build file currently supports even meson 0.49.0, so bumping the minimum aggressively may not be desired...

---

I would not be afraid to bump meson requirement to 0.57. I do not expect we will need to build it on some old systems, for example RHEL8 already has 0.58.

---

At any rate, permitting version 0.61.0 is sufficient to guarantee that people stuck on operating systems that only ship python 3.6 but desperately want bleeding edge OpenSC, can still manually acquire meson (by git cloning that too, or by using `pip install`) regardless of which version of meson is packaged by that distro repository.

The python requirement is a much harder one to ask people to manually compile the build system, but fortunately there are no more operating systems where python 3.5 is relevant. Even 3.6 is end of life and no longer receiving security updates by the Python Software Foundation (that's a baseline requirement for meson to drop support for an old version of python).